bootstrap/lib/aws.js

```diff
@@ -46,8 +48,12 @@ exports.initHandlerREST = (execute) => (event, _context, callback) => {
   if (!isContentTypeSupported(event)) {
     return callback(null, UNSUPPORTED_MEDIA_TYPE_RESPONSE)
   }
-
-  execute(event, (_, data) => {
+  let newBody = event.requestBody
+  newBody.data = Object.assign(
+    event.requestBody.data === undefined ? {} : event.requestBody.data,
+    parseQueryString(event.queryStringParameters),
+  )
+  execute(newBody, (_, data) => {
```

Comment:
Aha, now I get it :) You see how it confused me because now this code is coupled to some configuration that is contained in another repo. That other repo is actually not public, so this is going to be even a bigger issue for someone external to the team who would want to run this open-source software and be part of the Chainlink protocol.